tests/templates/kuttl/omid/00-limit-range.yaml

```
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: limit-request-ratio
+spec:
+  limits:
+    - type: "Container"
+      maxLimitRequestRatio:
+        cpu: 5
+        memory: 5
```

```suggestion
        memory: 1
```
In the other tests we use `1` to make sure we don't over-commit memory. It would be great if this would work here as well

---

Yes, we cannot do this here because Omid requires much more memory than all others

---

Ah, that's the ration between request and limit per container **not** between containers.